create_test_config.py

```diff
@@ -21,7 +21,7 @@
 def set_test_config(project, instance):
     config = configparser.ConfigParser()
     url = (
-        f"spanner:///projects/{project}/instances/{instance}/"
+        f"spanner+spanner:///projects/{project}/instances/{instance}/"
         "databases/compliance-test"
```

Comment:
This wasn't necessary in 1.3 version, but is required in 1.4. The change doesn't break 1.3 though.